### PR TITLE
Travis-CI

### PR DESCRIPTION
**Pull request by kevgo**
implements #890

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+go:
+  - 1.7
+
+rvm:
+  - 2.3.1
+
+
+before_install:
+  - git config --global user.email "you@example.com"
+  - git config --global user.name "Your Name"
```
Comment (charlierudolph) [inline .travis.yml]: Why are these needed? I'm guessing its because our feature test make commits in the test repos we setup but just wanted to check. If thats the case can we document why we need them.

Comment (kevgo) [inline .travis.yml]: Good catch, I removed them.
